### DISCUSSION TITLE
I am not seeing the source language of my app

### DISCUSSION THREAD

**Discussion by mouise1111** [dotnet/ResXResourceManager#518] [Q&A]
Hi there,

This is a bit of a silly question. But why am I not seeing the proper source language of my app? 
I am trying to auto translate from English to Dutch and French.
![image](https://user-images.githubusercontent.com/74969971/212500304-9c691cba-8a6b-4a1d-be61-6244e35d342a.png)
![image](https://user-images.githubusercontent.com/74969971/212500354-e6e77c97-c4e2-4f41-86c4-91766d2bdcfa.png)

**Comment by tom-englert** [comment]
Because you have the English resources in the Dutch resource file. They need to be in the neutral resource file.

**Reply by mouise1111** [reply]
Thanks, it worked!
